CONTRIBUTING.md

```
@@ -0,0 +1,259 @@
+| `feat` | New feature |
+| `fix` | Bug fix |
+| `docs` | Documentation changes |
+| `test` | Adding or updating tests |
+| `refactor` | Code change that neither fixes a bug nor adds a feature |
+| `style` | Formatting, missing semicolons, etc. |
+| `perf` | Performance improvement |
+| `chore` | Maintenance tasks |
+
+### Examples
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)